app/src/main/java/com/standuptracker/service/PersonService.kt

```diff
@@ -10,26 +10,27 @@ import retrofit2.Callback
 import retrofit2.Response
 import java.lang.Exception
 
+
 class PersonService {
     fun fetchPeople() : MutableLiveData<ArrayList<Person>> {
-        var _people = MutableLiveData<ArrayList<Person>>()
+        val people = MutableLiveData<ArrayList<Person>>()
```

Comment:
Best practice it to just name your variable with a lowercase. So that is why I named it as person instead of _person

Reply:
Half and half here.
- In Kotlin, when we have encapsulated variables (a public and a private version), the private version does start with a _
- But, variables declared in the scope of a function don't have access modifiers, so this one should not have a _.

If you are making changes to this variable name, ensure you've changed all references as well.

Reply:
manually changed.